src/lib/swapper/swappers/ThorchainSwapper/utils/getQuote/getQuote.ts

```diff
@@ -61,6 +62,7 @@ export const getQuote = async ({
     destination: parsedReceiveAddress,
     affiliate_bps: affiliateBps,
     affiliate: THORCHAIN_AFFILIATE_NAME,
+    streaming_interval: DEFAULT_STREAMING_INTERVAL,
```

Comment:
The interval is all we need to get a streaming swaps quote - swaps quantity can be omitted from the queryparams, meaning we let the network decide of the optimal swaps amount